lib/src/components/NxTabs/NxTab.tsx

```diff
@@ -40,16 +46,52 @@ const NxTab = function NxTabElement(props: NxTabProps) {
     }
   }
 
+  function handleKeyDown(event: React.KeyboardEvent<HTMLLIElement>) {
+    if (event.isDefaultPrevented()) {
+      return;
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const tabElements = event.currentTarget.parentElement!.children;
+
+    const getNextElementFromEventKey = (key: string) => {
+      switch (key) {
+        case 'Tab':
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          return event.currentTarget.parentElement!.nextSibling;
```

Comment:
This makes assumptions about the DOM structure that make me slightly nervous, since we rely on the calling code to arrange the `NxTabs` family of components correctly.  Conveniently however, we have access to the id of the tab panel via the `aria-controls` attribute of the current tab.  So I recommend using that to do a `document.getElementById` lookup.